packages/cli/src/constructs/browser-check.ts

```diff
@@ -17,6 +17,10 @@ export interface BrowserCheckProps extends CheckProps {
    * with the Puppeteer or Playwright frameworks.
    */
   code: Content|Entrypoint
+  /**
+   * A valid fully qualified domain name (FQDN) to check its SSL certificate.
```

Comment:
```suggestion
   * A valid fully qualified domain name (FQDN) to check for SSL certificate expiration. For example, 'app.checklyhq.com'.
```

Reply:
I think users might be a bit confused whether or not to add `https://`, etc. I think that adding a little hint could make sense.

Reply:
Totally agree. Thanks!